packages/webapp/pages/bookmarks/[folderId].tsx

```diff
@@ -33,10 +37,12 @@ const BookmarksPage = (): ReactElement => {
   const layout = getBookmarkFeedLayout(null, { seo }, props);
 
   useLayoutEffect(() => {
-    if (isReady && !isPending && !folder) {
+    const folderNotFound = !folder && isReady && !isPending;
```

Comment:
I think we can just show 404 than redirect actually. Similar to not found posts. Not a blocker though.

Reply:
I tried to change it quickly but returned ssr errors, maybe we should skip for now ok?